torch/nn/utils/rnn.py

```diff
@@ -248,6 +248,7 @@ def pad_sequence(sequences, batch_first=False):
         sequences (list[Variable]): list of variable length sequences.
         batch_first (bool, optional): output will be in BxTx* if True, or in
             TxBx* otherwise
+        padding_value (optional): values for padded elements.
```

Comment:
Can you make it equal to zero by default and always use `fill_`? It shouldn't make any difference in performance

Reply:
Also, please make the type description `(float, optional)`

Reply:
Yes, that makes more sense, but I was worried that it might make a difference. I'll take your word on it :)

For `float`, sure, but is it right? This should work for `LongTensors` and `0`, too, no?

Reply:
Hmm I guess so. It's still better to add some type hints.